core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -378,22 +378,22 @@ object TopicCommand extends Logging {
 
   def shortMessageSizeWarning(maxMessageBytes: Int): String = {
     "\n\n" +
-      "*****************************************************************************************************\n" +
-      "*** WARNING: you are creating a topic where the max.message.bytes is greater than the consumer ***\n" +
-      "*** default. This operation is potentially dangerous. Consumers will get failures if their        ***\n" +
-      "*** fetch.message.max.bytes < the value you are using.                                            ***\n" +
-      "*****************************************************************************************************\n" +
+      "******************************************************************************************************\n" +
+      "*** WARNING: you are creating a topic where the max.message.bytes is greater than the consumer     ***\n" +
```

Comment:
I added/removed some from some lines to make sure the lines in the message align nicely. It sounded to me that changes over time had broken the ending alignments.